firmware/robot2015/src-ctrl/scripts/linker/sections_FLASH.ld

```diff
@@ -0,0 +1,235 @@
+/* Linker script for mbed LPC1768 */
```

Comment:
Yeah it was the parameter linker script that never worked. I removed the target from the cmake builds, but I guess I never removed it from here...oops.